### PR TITLE
fix some typo

### PR DESCRIPTION
**Pull request by nguyenhoaidanh**

### CODE REVIEW

File: src/pages/Campaign/CampaignItem.tsx [hunk 1]
```diff
@@ -149,14 +149,14 @@ export default function CampaignItem({
             <ProgressBar
               title={t`Your Trading Volume`}
               percent={percentVolume}
-              value={`${tradingVolume}/${tradingVolumeRequired}`}
+              value={`${tradingVolume}/${tradingVolumeRequired}$`}
```
Comment (hungdoansy) [inline src/pages/Campaign/CampaignItem.tsx:152]: `$` should be the prefix, not suffix

Comment (nguyenhoaidanh) [reply src/pages/Campaign/CampaignItem.tsx:152]: Prakhar want it.
image

Comment (XiaoYhun) [reply src/pages/Campaign/CampaignItem.tsx:152]: its ok if passed QA test

**Review by XiaoYhun — APPROVED** [review]

File: src/pages/Campaign/CampaignItem.tsx (hunk 1 above)
Comment (hungdoansy) [reply src/pages/Campaign/CampaignItem.tsx:152]: Oh... Ok

**Review by hungdoansy — APPROVED** [review]

**Pull request merged**